### PR TITLE
Update gae

### LINKED ISSUE #167: Django Update for GAE

### PR DESCRIPTION
**Pull request by enriquejosepadilla**
fixes #167

### CODE REVIEW

File: django_cloud_deploy/workflow/__init__.py [hunk 1]
```diff
@@ -327,9 +329,13 @@ def update_project(self,
...
-        app_url = self._deploygke_workflow.update_app_sync(
-            project_id, cluster_name, django_directory_path,
-            django_project_name, image_name)
+        if backend == 'gke':
+            app_url = self._deploygke_workflow.update_app_sync(
+                project_id, cluster_name, django_directory_path,
+                django_project_name, image_name)
+        else:
+            app_url = self._deploygae_workflow.deploy_gae_app(
+                project_id, django_directory_path, False)
```
Comment (brianquinlan) [inline django_cloud_deploy/workflow/__init__.py]:
```suggestion
                project_id, django_directory_path, is_new=False)
```

File: django_cloud_deploy/workflow/_deploygae.py [hunk 2]
```diff
@@ -49,9 +51,10 @@ def deploy_gae_app(self,
         args = ['app', 'deploy', app_yaml_path, project]
         process = pexpect.spawn('gcloud', args)
         try:
-            index = process.expect(
-                ['\[{}\]\s*{}'.format(i, region) for i in range(1, 10)])
-            process.sendline(str(index))
+            if is_new:
```
Comment (brianquinlan) [inline django_cloud_deploy/workflow/_deploygae.py]: In a follow-up PR, could you just run gcloud with the '-q' flag to remove all this prompting stuff?

Comment (enriquejosepadilla) [reply django_cloud_deploy/workflow/_deploygae.py]: We tried, it doesn't work.

**Comment by enriquejosepadilla** [comment]
The suggestion added was by a Googler.

**Pull request merged**